hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OzoneFSInputStream.java

```diff
@@ -76,4 +81,39 @@ public boolean seekToNewSource(long targetPos) throws IOException {
   public int available() throws IOException {
     return inputStream.available();
   }
+
+  /**
+   * @param buf the ByteBuffer to receive the results of the read operation.
+   * @return the number of bytes read, possibly zero, or -1 if
+   *         reach end-of-stream
+   * @throws IOException if there is some error performing the read
+   */
+  @Override
+  public int read(ByteBuffer buf) throws IOException {
+
+    int bufInitPos = buf.position();
+    int readLen = Math.min(buf.remaining(), inputStream.available());
+
+    byte[] readData = new byte[readLen];
+    int bytesRead = inputStream.read(readData, bufInitPos, readLen);
+    buf.put(readData);
+
+    return bytesRead;
+  }
+
+  /**
+   * Query the stream for a specific capability.
+   *
+   * @param capability string to query the stream support for.
+   * @return True if the stream supports capability.
+   */
+  @Override
+  public boolean hasCapability(String capability) {
+    switch (StringUtils.toLowerCase(capability)) {
+    case "in:readbytebuffer":
```

Comment:
This case should be ```StreamCapabilities.READBYTEBUFFER``` that is supported in Hadoop 3.3.0 for now.
I use ```in:readbytebuffer``` here cause the dependency version of hadoop in Ozone is 3.2.0.

Reply:
Should we create a JIRA for this ?

Reply:
Sorry missed this, what is this change because Jira title says to support ByteBufferReadable.

Reply:
If it is something we want to support, can we address this in new Jira.